### ISSUE TITLE
Products.CMFCore requirement breaks when using CMFCore checkout

### ISSUE THREAD

**Issue by pbauer** [zopefoundation/Products.CMFUid#6]
`Products.CMFCore >= 2.4.0b3` is pinned which leads to:

```
While:
  Installing instance.
Error: There is a version conflict.
We already have: Products.CMFCore 2.4.0b3.dev0
but Products.CMFUid 3.0.dev0 requires 'Products.CMFCore>=2.4.0b3'.
```

**Comment by tseaver** [comment]
The pin should be easy to fix, no?  It should likely be `Products.CMFCore >= 2.4.0dev` until a CMFCore release is made.

**Comment by icemac** [comment]
Fixed in 3c9e3b8.
